assets/js/src/settings/pages/basics/default_sender.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import MailPoet from 'mailpoet';
+import { t, onChange } from 'settings/utils';
```

Comment:
Could we figure out how to solve this without creating utils? Utils will eventually end up as a file filled with unrelated stuff.
I see that it is convenient to have typed function for translations. What about creating some in common or maybe even on top-level directory so that others can use it?
For onChange I think we could create an input component with such behaviour. It may pass values and original event to the callback.
What do you think?